api/routes/analysis.py

```diff
@@ -337,19 +340,17 @@ async def get_proportionate_map(
     """
     pt = ProjectPermissionsTable(connection=connection.connection)
     project_ids = await pt.get_project_ids_from_names_and_user(
-        connection.author, projects, readonly=True
+        connection.author, list(projects), readonly=True
     )
 
     start_date = parse_date_only_string(start) if start else None
     end_date = parse_date_only_string(end) if end else None
 
     at = AnalysisLayer(connection)
-    results = await at.get_cram_size_proportionate_map(
+    return await at.get_cram_size_proportionate_map(
         projects=project_ids,
-        sequencing_types=sequencing_types,
+        sequencing_types=list(sequencing_types),
         start_date=start_date,
         end_date=end_date,
-        temporal_methods=temporal_methods,
+        temporal_methods=list(temporal_methods),
     )
```

Comment:
Ah I was trying to make the cache the calls to the propmap given that it's used quite frequently throughout the billing pages. I'm happy to revert since it wasn't very happy with the `@alru_cache` decorator even after making some datatype changes. It's not necessary so I'll probably just revert but may be worth thinking about doing at some point.